Packs/CommonTypes/ReleaseNotes/3_3_50.md

```diff
@@ -0,0 +1,4 @@
+
+#### Indicator Types
+
+- **emailRep**
```

Comment:
Please add an explanation about the changes.

Reply:
Roger